ledger/error/src/main/scala/com/daml/error/definitions/RejectionGenerators.scala

```diff
@@ -3,15 +3,22 @@
 
 package com.daml.error.definitions
 
-import com.daml.error.{BaseError, ContextualizedErrorLogger, ErrorCause}
+import com.daml.error.{BaseError, ContextualizedErrorLogger}
 import com.daml.lf.engine.Error.{Interpretation, Package, Preprocessing, Validation}
 import com.daml.lf.engine.{Error => LfError}
 import com.daml.lf.interpretation.{Error => LfInterpretationError}
 import io.grpc.StatusRuntimeException
 
+sealed abstract class ErrorCause extends Product with Serializable
```

Comment:
I am curious what was the reason of changing sealed trait to sealed abstract class?

Reply:
I wasn't what I did :)
I removed `ErrorCauseExport` and moved pre-existing `ErrorCause` here.

Reply:
I see, it just ErrorCause is very similar to old removed ErrorCauseExport.. but still `sealed abstract class` is confusing as there is nothing in this class which would make me think it is abstract class rather than trait :)